.github/workflows/node.js.yml

```diff
@@ -29,3 +29,23 @@ jobs:
     - run: npm ci
     - run: npm test
     - run: npm run eslint
+
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
Is it possible to define `@latest` ?

Reply:
From the documentation I checked, it doesn't seem to be possible.
https://github.com/orgs/community/discussions/25772

But we have dependabot which will detect updates and apply them.

Reply:
I'm happy with this alternative approach!